sample/list.json

```diff
@@ -24,6 +24,14 @@
     "type": "wasm"
   },
   {
+    "name": "[NaCl] Hello World",
+    "description": "Convert 'Hello World' NaCl sample to WASM",
+    "src": "HelloWorld_NaCl",
+    "icon": "icon.png",
+    "language": "C++",
+    "type": "wasm"
```

Comment:
Is it correct the type is wasm ?

Reply:
Yes. The NaCl sample is needed to compile source to WASM.
So I set the 'wasm' type.